apps/samples/Ensemble8/src/app/java/ensemble/samplepage/Description.java

```diff
@@ -31,26 +31,28 @@
  */
 package ensemble.samplepage;
 
-import ensemble.EnsembleApp;
-import ensemble.PlatformFeatures;
-import ensemble.SampleInfo;
-import ensemble.SampleInfo.URL;
-import ensemble.generated.Samples;
+import static ensemble.samplepage.SamplePage.INDENT;
+import static ensemble.samplepage.SamplePageContent.title;
+import javafx.application.ConditionalFeature;
+import javafx.application.Platform;
 import javafx.event.ActionEvent;
-import javafx.event.EventHandler;
 import javafx.geometry.Insets;
-import javafx.scene.control.*;
+import javafx.scene.control.ContentDisplay;
+import javafx.scene.control.Hyperlink;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.Tooltip;
 import javafx.scene.image.Image;
 import javafx.scene.image.ImageView;
 import javafx.scene.layout.ColumnConstraints;
 import javafx.scene.layout.GridPane;
 import javafx.scene.layout.VBox;
 import javafx.scene.text.Text;
-import javafx.util.Callback;
-import static ensemble.samplepage.SamplePage.INDENT;
-import static ensemble.samplepage.SamplePageContent.title;
-import javafx.application.ConditionalFeature;
-import javafx.application.Platform;
+import ensemble.EnsembleApp;
+import ensemble.PlatformFeatures;
+import ensemble.SampleInfo;
+import ensemble.SampleInfo.URL;
+import ensemble.generated.Samples;
```

Comment:
I see that `ensemble.xxx` is moved after `javafx.xxx`; similarly, elsewhere I see `com.sun.xxx` imports moved after. What's the algorithm your IDE uses to sort imports? sort stuff that starts with "java" first and then everything else alphabetically?

Reply:
this is how the IDE formatter is currently configured - it should not matter, especially since it dos not use wildcards (except for static imports).

Reply:
It looks like Eclipse uses the following order **by default**: [java,javax,org,com].